### ISSUE TITLE
Pure Dark Mode

### ISSUE THREAD

**Issue by CodeNomadHQ** [PhilKes/NotallyX#16]
**Describe the solution you'd like**
The app currently uses a mix of gray and dark colors, but I think it would be great to have a pure dark mode option.

**Comment by alekksander** [comment]
Yes, please make it possible to ditch grayish tone in app and have pitch black instead.

**Comment by andiandi13** [comment]
I second that.
Being on OLED screen, it'd save a lot of battery by turning off the LED if it's pitch black 👍

**Comment by PhilKes** [comment]
> Yes, please make it possible to ditch grayish tone in app and have pitch black instead

> I second that. Being on OLED screen, it'd save a lot of battery by turning off the LED if it's pitch black 👍

Just so that I understand because I dont have any OLED screen, are you guys refering to the background color not being exactly `#000000`?

**Comment by andiandi13** [comment]
> > Yes, please make it possible to ditch grayish tone in app and have pitch black instead
> 
> > I second that. Being on OLED screen, it'd save a lot of battery by turning off the LED if it's pitch black 👍
> 
> Just so that I understand because I dont have any OLED screen, are you guys refering to the background color not being exactly `#000000`?

Yes. on oled when background or any part of the screen is #000000, diodes are simply turned off, which saves a lot of battery, contrary to a LCD screen where backlight is always on.

**Comment by SagXD** [comment]
> > Yes, please make it possible to ditch grayish tone in app and have pitch black instead
> 
> > I second that. Being on OLED screen, it'd save a lot of battery by turning off the LED if it's pitch black 👍
> 
> Just so that I understand because I dont have any OLED screen, are you guys refering to the background color not being exactly `#000000`?
> 
> 
> 
> 

Something like this:

![output-onlinetools.png](https://github.com/user-attachments/assets/66b05428-cae6-4a00-8207-b765db52be72

**Comment by andiandi13** [comment]
Personally I prefer the dark grey dialogs. 
And in most apps I use that have a pure dark mode, dialog boxes are in some kind of grey for more lisibility.

I find it a bit confuse on left image

**Comment by andiandi13** [comment]
Here are some examples :

(Namely Vanilla music, MX Player, NewPipe, Flym, and QP Gallery mod)

![Image](https://github.com/user-attachments/assets/86972863-0e95-4807-b90e-0a5db49494f4)
![Image](https://github.com/user-attachments/assets/5e497cba-6789-4c2f-b203-45dcef7635c8)
![Image](https://github.com/user-attachments/assets/6238665b-9463-4b99-ba9a-daaea7968975)

**Comment by SagXD** [comment]
Something like this: 

![Screenshot_20250131-090802.png](https://github.com/user-attachments/assets/e7fe56cb-baba-4467-b597-6031dd8aeb08)

Moshidon client for Mastodon

**Comment by SagXD** [comment]
> I am currently working on this, changing the dark mode's black colors to pitch black
> 
> One thing I was not sure about is dialogs, do you guys want them to be pitch black too? By default they are lightened up by the Material Theme to improve readability:
> 
> <p>
> <img src="https://github.com/user-attachments/assets/1249a1a2-bff0-4b87-bbcb-1930767da298" width=300 />
> <img src="https://github.com/user-attachments/assets/d08445d5-de21-4624-824c-163e186c3dea" width=300 />
> </p>
> 
> I personally dont care much, but I would let you guys decide on it.


Second Image is better.
